dashboard/app/views/plc/learning_modules/new_learning_resource_for_module.html.haml

```diff
@@ -0,0 +1,37 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/levelbuilder', media: 'all'
+
+%h3{style: 'margin-bottom: 24px;'}
+  Add a resource (Module: #{@learning_module.name})
+
+.new_resource_section{style: 'float: left; width: 33%'}
+  %p
+    = link_to 'New Script', new_script_path, class: 'btn'
+  %p
+    Select Script if
+    %li
+      You want to create multiple screens that a learner will click through
+    %li
+      You will incorporate multiple puzzles or questions
+
+.new_resource_section{style: 'float: right; width: 33%'}
+  %p
+    = link_to 'New Link', new_plc_task_path(plc_learning_module_id: @learning_module.id, type: 'Plc::LearningResourceTask'), class: 'btn'
```

Comment:
I don't totally understand what happens when I click this link.  The title of the page says "New plc_task".  Am I adding a resource or a task?  (Are those the same thing?)